tutorcairn/templates/cairn/apps/superset/superset_config.py

```diff
@@ -18,6 +18,10 @@
 # https://superset.apache.org/docs/installation/configuring-superset/#configuration-behind-a-load-balancer
 ENABLE_PROXY_FIX = True
 
+{% if not ENABLE_HTTPS %}
```

Comment:
What's talisman? Can you please add a comment to explain what it is, maybe with a link?